src/components/view/WithFudgyClick.tsx

```diff
@@ -0,0 +1,31 @@
+import { GroupProps } from '@react-three/fiber'
+import { useState } from 'react'
+
+const LEFT_MOUSE_BUTTON = 0
+
+/**
+ * HOC that adds "fudgy" click events that can be handled via an `onFudgyClick` prop.
+ * If this HOC is called in a functional component, consider memoization.
```

Comment:
Can you add a bit of elaboration - I am not sure what a fudgy click is

Reply:
there used to be an old comment that said "add some fudge factor to the mouse coordinates". It allows for the slightest of pointer movements between mouse down and mouse up.